BlenderRenderController/BlenderRenderController/MainForm.cs

```diff
@@ -172,20 +245,148 @@ private void concatenatePartsButton_Click(object sender, EventArgs e)
+			p.StartInfo.UseShellExecute        = false;
+
+            p.StartInfo.Arguments = String.Format("-b \"{0}\" -P \"{1}\"",
+                                                  blendFilePathTextBox.Text,
+                                                  Path.Combine(ScriptsPath, "get_project_info.py")
+                                    );
+
+			try {
+				p.Start();
+			}
+			catch( Exception ex ) {
+				Trace.WriteLine( ex );
+			}
+
+			StringBuilder jsonInfo    = new StringBuilder();
+			bool          jsonStarted = false;
+			int           curlyStack  = 0;
+
+			while( !p.StandardOutput.EndOfStream ) {
+				string line = p.StandardOutput.ReadLine();
+
+				if( line.Contains( "{" ) ) {
+					jsonStarted = true;
+					curlyStack++;
+				}
+
+				if( jsonStarted ) {
+
+					if( !line.ToLower().Contains( "blender quit" ) && curlyStack > 0 ) {
+						jsonInfo.AppendLine( line );
+					}
+
+					if( line.Contains( "}" ) ) {
+
+						curlyStack--;
+
+						if( curlyStack == 0 ) {
+							jsonStarted = false;
+						}
+
+					}
+
+				}
+
+			}
+
+			BlenderData blendData = null;
+			if( jsonInfo.Length > 0 ) { 
+				JavaScriptSerializer serializer = new JavaScriptSerializer();
+				blendData = serializer.Deserialize<BlenderData>( jsonInfo.ToString() );
+			}
+
+			if( blendData != null ) {
+				startFrameNumericUpDown.Value      = blendData.StartFrame;
+				totalFrameCountNumericUpDown.Value = blendData.EndFrame;
+				outFolderPathTextBox.Text          = outFolderPath = blendData.OutputDirectory;
+				//blendProjectName                   = blendData.ProjectName;
+
+				if( blendData.EndFrame < endFrameNumericUpDown.Value ) {
+					endFrameNumericUpDown.Value = blendData.EndFrame;
+				}
+
+			}
+
+			Trace.WriteLine( "Json data = " + jsonInfo.ToString() );
+
+			//Trace.WriteLine( String.Format( "CEW: {0}", p.StartInfo.Arguments ) );
+
+		}
+
+		private void ReadBlenderData_Click( object sender, EventArgs e ) {
+			DoReadBlenderData();
+		}
+
+		private void MixdownAudio_Click( object sender, EventArgs e ) {
+
+			if( !File.Exists( blendFilePathTextBox.Text ) ) {
+				return;
+			}
+
+			if( !Directory.Exists( outFolderPathTextBox.Text ) ) {
+				Directory.CreateDirectory( outFolderPathTextBox.Text );
+			}
+
+            Process p = new Process();
+
+            p.StartInfo.FileName               = "blender";
+			p.StartInfo.RedirectStandardOutput = true;
+			p.StartInfo.UseShellExecute        = false;
+			//Using minimized instead so we get feedback
+			//p.StartInfo.CreateNoWindow         = true;
+			p.StartInfo.WindowStyle = ProcessWindowStyle.Minimized;
+
+            p.StartInfo.Arguments = String.Format("-b \"{0}\" -P \"{1}\"",
+                                                  blendFilePathTextBox.Text,
+                                                  Path.Combine(ScriptsPath, "mixdown_audio.py")
+                                    );
+
+			p.Start();
+
+			p.WaitForExit((int)TimeSpan.FromMinutes(5).TotalMilliseconds);
```

Comment:
Thank you for your very thorough explanation! :)
It was my mistake overlooking the fact of that code being not in the video rendering part, but rather in the audio mixing.

I have too, sometimes noticed clicky audio, but wasn't sure what to attribute it to. It's very nice that you've been able to track down that issue!

I feel like you know much more about this whole thing than I do, so I am very thankful for your contributions!

I am now merging this PR and editing my StackExchange answer to make sure, you are mentioned there! ;)